scripts/ecnconfig

```diff
@@ -84,11 +115,73 @@ class EcnConfig(object):
             print("Setting %s value to %s%%" % (field, value))
         self.db.mod_entry(WRED_PROFILE_TABLE_NAME, profile, {field: value})
 
+class EcnQ(object):
+    """
+    Process ecn on/off on queues
+    """
+    def __init__(self, queues, verbose):
+        self.ports_key = ''
+        self.queues = queues.split(',')
+        self.validate_queues()
+        self.verbose = verbose
+
+        # Set up db connections
+        self.config_db = swsssdk.ConfigDBConnector()
+        self.config_db.connect()
+
+        self.db = swsssdk.SonicV2Connector(host="127.0.0.1")
```

Comment:
>host [](start = 43, length = 4)

We should use unix socket to improve performance. I notice many cases in this repo.

Reply:
Is there a way to connect to unix socket via SonicV2Connector?  I need the get() member function to get the value of a field. ConfigDBConnector does not provide an equivalent call; its get_entry() returns a list of field-value tuples under a key, not a value of a particular field. I replace the set() call by ConfigDBConnector's mod_entry() to minimize the usage through ConfigDBConnector